src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -253,9 +254,18 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             var response = new WorkerInitResponse
             {
                 Result = new StatusResult { Status = StatusResult.Types.Status.Success },
-                WorkerVersion = WorkerInformation.Instance.WorkerVersion
+                WorkerVersion = WorkerInformation.Instance.WorkerVersion,
+                WorkerMetadata = new WorkerMetadata
+                {
+                    RuntimeName = RuntimeInformation.FrameworkDescription,
+                    RuntimeVersion = Environment.Version.ToString(),
+                    WorkerVersion = WorkerInformation.Instance.WorkerVersion,
+                    WorkerBitness = RuntimeInformation.ProcessArchitecture.ToString()
+                }
             };
 
+            response.WorkerMetadata.CustomProperties.Add("Grpc.Worker.Version", typeof(GrpcWorker).Assembly.GetName().Version?.ToString());
```

Comment:
The package name ends with `Worker.Grpc`. Should we use that(`Worker.Grpc.Version`) here instead of `Grpc.Worker.Version`?

Reply:
Should we also include Worker.Sdk version?

Do we care about the Worker.Core package version?

Reply:
Oh, I see Worker.Core version is in the `WorkerVersion` property.

Reply:
> Member

That's a good question. I don't really have a strong preference for this.

Reply:
> Should we also include Worker.Sdk version?

Could be interesting, though, we don't really have any artifacts injected today. Might be a good thing to add to the target assembly and then we could update this code to use that information.

> Do we also care about getting the Worker.Core package version?

Saw your note below. Yes, this is already tracked.